docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -217,7 +217,7 @@ In other words, if one were to print out the compressed data as
 a sequence of bytes, starting with the first byte at the
 *right* margin and proceeding to the *left*, with the most-
 significant bit of each byte on the left as usual, one would be
-able to parse the result from right to left, with fixed-width
+able to parse the result from right-to-left, with fixed-width
```

Comment:
I don't think you need dashes, since the expression here is "from right to left".